src/core/env/src/main/scala/EnvironmentUtils.scala

```diff
@@ -18,7 +18,7 @@ object EnvironmentUtils {
     val nvsmicmd = if (IsWindows) {
       // Unlikely nvidia is on the path
       val nvsmi = Paths.get(
-        System.getenv("ProgramFiles"),
+        sys.env.getOrElse("ProgramFiles", null),
```

Comment:
this should just be sys.env.get to keep with the same semantics. The old code threw an error if program files were not found this will fail silently.

Reply:
wanted to have same behaviour on this line as System.getenv - returning value or null, but I can change it to throw exception if variable not found (java.util.NoSuchElementException)